ui/darksky/README.md

```diff
@@ -57,7 +63,7 @@ The next two steps are completed differently based on the version of HA you are
 - Pre 0.84 or if using yaml mode in 0.84 or above : Add to your ui-lovelace.yaml file.
 - Using storage mode in 0.84 or above use the "Raw Config Editor" to add the reference and definition to the config.
 
-2. Add the card reference at the top of the configuration ***Note the use of type: module and not js
+2. Add the card reference at the top of the configuration ***Note the use of type: module and not js **Note2: /local/ points to the /config/www/ dir.
```

Comment:
~~~~
 **Note2: /local/ points to the /config/www/ dir.
~~~~
should follow same convention as your other edits. eg:
~~~~
 **Note2: /local/ points to the <config-dir>/www/ dir.
~~~~